Source/WTF/wtf/text/StringConcatenate.h

```diff
@@ -44,6 +44,13 @@ WTF_EXPORT_PRIVATE extern std::atomic<int> wtfStringCopyCount;
 
 namespace WTF {
 
+class StringBuilder;
+
+/// A type is `StringTypeAdaptable` if there is a specialization of `StringTypeAdapter` for that type.
```

Comment:
What does the third slash mean?

Love the thoughtful use of concepts.

Reply:
Ah, this is what other projects use for more documentation style comments and headerdocs, and my mind cross-contaminating styles.

This should be // in WTF.